website/_toc.yml

```diff
@@ -0,0 +1,48 @@
+- file: landing-page
+
+- part: "User documentation"
+  chapters:
+    - file: user-documentation/getting-started
+    - file: user-documentation/introduction
+    - file: user-documentation/molecules
+    - file: user-documentation/orbital-bases
+    - file: user-documentation/quantum-chemical-methods
+    - file: user-documentation/quantum-chemical-models
+    - file: user-documentation/use-cases
+
+- part: "Examples"
+  chapters:
+    - file: examples/AP1roG-calculations
+    - file: examples/CCSD
+    - file: examples/Custom-algorithms-showcase-CUHF
+    - file: examples/DOCI
```

Comment:
Including this `DOCI` notebook results in a confusing table of contents:

![Screenshot 2020-12-09 at 10 12 22](https://user-images.githubusercontent.com/33431368/101609437-42831100-3a07-11eb-9a1f-5046955c1227.png)

Reply:
"Preparing the Hamiltonian in the canonical RHF basis", "Dense DOCI calculations" and "Davidson DOCI calculations" are all sub parts of DOCI calculations and should not be present in the TOC in my opinion.